modules/manage/partials/authentication.adoc

```diff
@@ -0,0 +1,1583 @@
+  --set config.cluster.oidc_token_expire_disconnect=<true-or-false> \
+  --set config.cluster.oidc_keys_refresh_interval=<interval>
+----
+
+====
+--
+======
+
+- `config.cluster.sasl_mechanisms`: Enable SCRAM and OIDC SASL mechanisms.
+- `config.cluster.oidc_discovery_url`: The discovery URL of your identity provider (IdP). The default is `https://auth.prd.cloud.redpanda.com/.well-known/openid-configuration`.
+- `config.cluster.oidc_token_audience`: The intended recipient of the token. The default is `redpanda`.
+- `config.cluster.oidc_principal_mapping`: The principal mapping, which is a JSON path that extracts a principal from any claim in the access token payload. The default is `$.sub`.
+- `config.cluster.oidc_clock_skew_tolerance`: The amount of time (in seconds) to allow for when validating the expiration claim in the token.
+- `config.cluster.oidc_token_expire_disconnect`: Whether to enable OIDC to disconnect clients when their token expires.
+- `config.cluster.oidc_keys_refresh_interval`: The amount of time keys from the `jwks_uri` are cached.
+endif::[]
+ifndef::env-kubernetes[]
+. Enable the `OAUTHBEARER` SASL mechanism:
++
+[,bash]
+----
+rpk cluster config set sasl_mechanisms '["SCRAM","OAUTHBEARER"]' -X admin.hosts=localhost:9644
+----
++
+Example output:
++
+[.no-copy]
+----
+Successfully updated configuration. New configuration version is 16.
+----
+
+. Specify the discovery URL of your identity provider (IdP). The following IdP
+URL uses the default value:
++
+[,bash]
+----
+rpk cluster config set oidc_discovery_url 'https://auth.prd.cloud.redpanda.com/.well-known/openid-configuration'
+----
++
+This sets the OIDC-specific configuration
+for all clusters in the `.bootstrap.yaml` (located in the same directory as the `redpanda.yaml`). The OIDC
+discovery URL is read upon the first startup after configuration, and ignored on all subsequent startups.
+Use `rpk cluster config set` for any subsequent modifications of the discovery URL.
+
+. Specify the intended recipient of the token:
++
+[,bash]
+----
+rpk cluster config set oidc_token_audience 'redpanda'
+----
+
+. Specify the principal mapping, which is a JSON path that extracts a principal from any claim in the access token payload:
++
+[,bash]
+----
+rpk cluster config set oidc_principal_mapping '$.sub'
+rpk cluster config set oidc_principal_mapping '$.email'
+----
+
+. Specify the amount of time (in seconds) to allow for when validating the expiration claim in the token:
++
+[,bash]
+----
+rpk cluster config set oidc_clock_skew_tolerance 30
+----
+
+. Enable OIDC to disconnect clients when their token expires:
++
+[,bash]
+----
+rpk cluster config set oidc_token_expire_disconnect true
+----
+
+. Specify the amount of time keys from the `jwks_uri` are cached:
++
+[,bash]
+----
+rpk cluster config set oidc_keys_refresh_interval 3600
+----
+endif::[]
+
+If you're configuring authentication for the first time, you may need to
```

Comment:
Shouldn't this come very early on in the section/topic--before configuring authN?